CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ set(FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB "49" CACHE STRING
     "Size of the command-stream buffer. As a rule of thumb use the same value as FILAMENT_PER_FRRAME_COMMANDS_SIZE_IN_MB, default 1."
 )
 
-set(FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB "10" CACHE STRING
+set(FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB "100" CACHE STRING
```

Comment:
nit: Would it make sense to set it to 96 as in Metal backend? As far as I remember, @marcisolti experimented with Metal handle arena size, and found that 96 is the sweet spot there.

Reply:
Metal handles have twice as much memory footprint than OpenGL handles (GLuint vs pointers), but on Windows we expect our users to work with much larger workspaces (more handles), and it doesn't help either that we create buffer objects for each part, so I'm not even sure, that the 100Mb is enough on the long run, but we can change.